src/snowflake/connector/_sql_util.py

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (c) 2012-2021 Snowflake Computing Inc. All rights reserved.
+#
+
+from __future__ import annotations
+
+import re
+
+from .constants import FileTransferType
+
+COMMENT_START_SQL_RE = re.compile(
+    r"""
+                                  ^\s*(?:
+                                      /\*[\w\W]*?\*/
+                                  )""",
```

Comment:
Why do we need these new lines and spaces in this regex?

Reply:
It's easier to read. Here's the doc on [re.VERBOSE](https://docs.python.org/3/library/re.html#re.VERBOSE)